amplitude_python_sdk/v2/client.py

```diff
@@ -17,10 +19,16 @@ class AmplitudeV2APIClient:  # pylint: disable=too-few-public-methods
     """
 
     def __init__(
-        self, api_key: str, api_endpoint: str = "https://api2.amplitude.com/2"
+        self,
+        api_key: str,
+        secret_key: str,
+        api_endpoint: str = "https://api2.amplitude.com/2",
+        api_endpoint_2: str = "https://amplitude.com/api/2"
```

Comment:
I think this is pretty murky as to what it really means - let's name this something like `chart_annotations_api_endpoint` for now, and we can rename later if need be for other endpoints.

Reply:
Done